pagefaultgames/pokerogue
# [Bug] i18n messages files fix

> [!TIP]
> https://github.com/pagefaultgames/pokerogue-locales/pull/21

## Why am I making these changes?

So apparently Pontoon has this special behavior where it expects any files containing the word `messages` to have a specific kind of json pattern.

So instead of adapting to that I just decided to rename those files. Since we don't want to change their namespaces i had to add them to namespace->file matcher.

## What are the changes from a developer perspective?

Make sure that the namespaces for `*messages` get matched to the correctly renamed files

## How to test the changes?

Run the game
see if the renamed files are loaded properly so no i18n strings are missing.

## Checklist
